mdal/frmts/libplyxx/libplyxx.h

```diff
@@ -0,0 +1,226 @@
+#pragma once
```

Comment:
I was expecting and understand the concerns. I would make the following points :

- I personally would not count this as including the library as much as including code forked from that repository. I have extensively updated the code to integrate into MDAL, to add List Properties, to complete the set of data types that are supported and to fix a number bugs that did exist. So much so that, even if the source repository were actively managed this fork is now so far from the source that it can no longer take updates from the source. This is a self-contained driver that has to be viewed and maintained as stand-alone, that happens to include code (legally) forked from the original source - more of a template than a library.

- However, the basic structure using templates, function abstraction and maps, is sound and using that has saved considerable amounts of time putting the basic structure together. It is this that we are getting for free. Despite that, as part of the upgrades mentioned above, I have been through every line of the code, know what it does etc. There are no functions included that are not exclusively used for the process of reading or writing ASCII and BINARY PLY files and the same level of complexity would be required for a dedicated built-from-scratch driver, and the module design that this code brings makes the actual integration to the MDAL data model very simple, clear and flexible.

- I would note that the previous (read-only) driver was (at your insistence actually) copyright Runette Software. Legally (and ethically) this new driver is no different. It is code included into MDAL that is (from your point of view) 3rd party copyright. The only thing that has changed is the owner of the copyright. And the namespace - but that can be changed if it is a problem.

- Note that there are no "standard libraries" for reading PLY files. I did look. There are published example implementations - including Greg Turk's original - but actually, none of them is published under a licence that could be used as part of MDAL.

Reply:
OK to include it as it. But lets only add to the header of the files from libply++ that the file is `forked from the repo XYZ with MIT Licence Copyright Simon ...` Also I would not stick with the original names, lets rename them so it makes sense in context of MDAL.

Reply:
@runette 
> Note that there are no "standard libraries" for reading PLY files. I did look. There are published example implementations - including Greg Turk's original - but actually, none of them is published under a licence that could be used as part of MDAL.

There is https://github.com/phma/plytapus under a MIT license. (Not sure how mature it is though)